lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -660,6 +659,123 @@ def get(self):
             folderIndex[groups].items.append(attributeInfo.name)
         return self._attributeLayout
 
+class DisplayCustomDataControl(object):
+    '''Custom control for adding in extra display custom data.'''
+    def __init__(self, item, prim):
+        super(DisplayCustomDataControl, self).__init__()
+        self.item = item
+        self.prim = prim
+
+        # Check whether Ufe has the scene item meta data support.
+        self.useMetadata = hasattr(ufe.SceneItem, "getGroupMetadata") and hasattr(ufe.Value, "typeName")
+
+    @property
+    def GROUP(self):
+        # When using meta data we use a special group name which will automatically
+        # write to the session layer.
+        return 'SessionLayer-Autodesk' if self.useMetadata else 'Autodesk'
```

Comment:
When metadata we use the SceneItem method which has the special group name to target the SessionLayer. Otherwise when using USD to save custom data we will target SessionLayer ourself.